docs/feature_split_keyboard.md

```diff
@@ -89,7 +89,13 @@ You can configure the firmware to read a pin on the controller to determine hand
 #define SPLIT_HAND_PIN B7
 ```
 
-This will read the specified pin. If it's high, then the controller assumes it is the left hand, and if it's low, it's assumed to be the right side. 
+This will read the specified pin. By default, if it's high, then the controller assumes it is the left hand, and if it's low, it's assumed to be the right side. 
+
+This behaviour can be flipped by adding this to you `config.h` file:
+
+```c
+#define	SPLIT_HAND_LOW_IS_LEFT
```

Comment:
```suggestion
#define	SPLIT_HAND_PIN_LOW_IS_LEFT
```

Reply:
I chose `SPLIT_HAND_PIN_IS_LEFT` to match `SPLIT_HAND_GRID_IS_RIGHT` (compared with the `SPLIT_HAND_GRID` define).

I'm happy to make the requested change, but it does not feel consistent with the other defines used for the same functionality

Reply:
You might want to double check what you have submitted, as it wouldnt show suggestions if it were `SPLIT_HAND_PIN_LOW_IS_LEFT`

This alignment suggestion is to `<handedness-option>_LOW_IS_<hand>` so that it matches the existing `SPLIT_HAND_MATRIX_GRID` and `SPLIT_HAND_MATRIX_GRID_LOW_IS_RIGHT`

Reply:
you're right, I remembered my intention, but not that I'd done it wrong :)